policyengine_canada/variables/gov/provinces/qc/tax/income/credits/fa/qc_fa_eligibility.py

```diff
@@ -0,0 +1,19 @@
+from policyengine_canada.model_api import *
+
+
+class qc_fa_eligibility(Variable):
+    value_type = float
+    entity = Household
+    label = "Quebec family allowance eligible indivudual"
+    reference = "https://www.legisquebec.gouv.qc.ca/en/document/cs/I-3?langCont=en#se:1029_8_61_8"
+    definition_period = YEAR
+    defined_for = ProvinceCode.QC
+
```

Comment:
This will get properly summed and cast to bool. We don't need parameters.
```suggestion
    adds = ["own_children_in_household"]
```